runtime/proto/service/log_replication.proto

```diff
@@ -52,4 +54,36 @@ enum LogReplicationEntryType {
   SNAPSHOT_REPLICATED = 4;
   SNAPSHOT_END = 5;
   SNAPSHOT_TRANSFER_COMPLETE = 6;
-}
\ No newline at end of file
+}
+
+enum ReplicationModel {
+  FULL_TABLE = 0;                 // Full table replication (Single Source to Single Sink = 1:1)
```

Comment:
Hi Shreay, remember to remove the protobuf message defined in infra/ module, otherwise we will have 2 duplicate messages

Reply:
I think for this I'll create a small separate pr since there are some dependencies in lrv2